config/routes.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'subjects/index'
```

Comment:
this should be removed, because it is duplicate with  `resources :subjects, only: %i[index]`